### PR TITLE
do local development in Docker

### PR DESCRIPTION
**Pull request by afeld**
Copied in config files from the Handbook.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+FROM ruby:2.6.1
+
+# Bundle install first for a simple gem cache
+COPY Gemfile* /tmp/
+WORKDIR /tmp
+RUN bundle install
+
+# Set app working direcotry and copy app there.
```
Comment (adunkman) [inline Dockerfile]: `direcotry` is misspelled 😇

**Review by adunkman — APPROVED** [review]
Looks good! 👍

**Comment by siennathesane** [comment]
Since you asked for feedback, looking at your dockerfile, if you don't need the libraries outside of compilation, you might as well do a multi-stage build: https://docs.docker.com/develop/develop-images/multistage-build/. Or you could use a buildpack (same thing cloud.gov does): https://buildpacks.io/

**Comment by afeld** [comment]
> if you don't need the libraries outside of compilation, you might as well do a multi-stage build

Not sure that's worth the effort / would have much benefit since this is just for development. In any case, it works as is so am going to go ahead and merge, but pull requests for optimizations are welcome.

> you could use a buildpack

Yeah, have been curious to try this locally but haven't gotten around to it. Ditto the above.

**Pull request merged**